doc/deps.md

```diff
@@ -109,28 +123,34 @@ cd libodb-2.4.0
 make install
 cd ..
 
-# If you use SQLite
+#
+# If you use SQLite:
+#
+sudo apt-get install libsqlite3-dev # Needed for this step.
 wget http://www.codesynthesis.com/download/odb/2.4/libodb-sqlite-2.4.0.tar.gz
 tar -xvf libodb-sqlite-2.4.0.tar.gz
 cd libodb-sqlite-2.4.0
-./configure --prefix=<odb_install_dir>
+./configure --prefix=<odb_install_dir> \
+  --with-libodb="$(readlink -f ../libodb-2.4.0)"
 make install
 cd ..
 
-# If you use PostgreSQL
-sudo apt-get install postgresql-server-dev-<version> # Needed for this step
+#
+# If you use PostgreSQL:
+#
+sudo apt-get install postgresql-server-dev-<version> # Needed for this step.
 wget http://www.codesynthesis.com/download/odb/2.4/libodb-pgsql-2.4.0.tar.gz
 tar -xvf libodb-pgsql-2.4.0.tar.gz
 cd libodb-pgsql-2.4.0
-./configure --prefix=<odb_install_dir>
+./configure --prefix=<odb_install_dir> \
+  --with-libodb="$(readlink -f ../libodb-2.4.0)"
 make install
 cd ..
 
-sudo apt-get install gcc-<version>-plugin-dev
-sudo apt-get install libcutl-dev
+sudo apt-get install gcc-<version>-plugin-dev libcutl-dev libexpat1-dev
```

Comment:
Why is `libexpat1-dev` needed?

Reply:
`libexpat1-dev` is the header/development package for `libexpat1` which is a dependency of `libcutl` which is the header/development package for `libcutl-dev`. Without installing this, some compilation fails due to missing headers or linker errors.

Most likely, considering this install command has been added to this line, it's the ODB command-line executable that cannot compile without it.